src/state/shellscripts/sandbox_exec.sh

```diff
@@ -49,9 +49,9 @@ add_ccache_mount() {
 }
 
 add_dune_cache_mount() {
-  DUNE_CACHE=${XDG_CACHE_HOME:-$HOME/.cache}/dune
-  mkdir -p "${DUNE_CACHE}"
-  add_mounts rw "$DUNE_CACHE"
+  dune_cache=${XDG_CACHE_HOME:-$HOME/.cache}/dune
```

Comment:
```suggestion
  local dune_cache=${XDG_CACHE_HOME:-$HOME/.cache}/dune
```

Reply:
Do we need to do this for non-macOS system script? (e.g. `src/state/shellscripts/bwrap.sh`)

Reply:
It's cleaner yes

Reply:
Done.